apps/dashboard/src/components/WorkspaceTable.tsx

```diff
@@ -198,7 +198,12 @@ export function WorkspaceTable({
               !loading && (
                 <TableRow>
                   <TableCell colSpan={columns.length} className="h-24 text-center">
-                    No results.
+                    <div className="flex flex-col items-center justify-center space-y-2">
+                      <p className="text-muted-foreground">You don't have any Sandboxes yet.</p>
+                      <p className="text-sm text-muted-foreground">
+                        Create your first sandbox to get started with isolated development environments.
```

Comment:
Since we have the onboarding info above the table, when it's empty, I suggest removing this